2009-10-28  Stepan Kasal  <skasal@redhat.com>

	* aclocal.m4 (BRLTTY_SUMMARY_END): Use single quotes when
	passing the value of brltty_summary_lines, so that is not
	garbled.

--- brltty-4.1/aclocal.m4.orig	2009-10-08 04:32:49.000000000 +0200
+++ brltty-4.1/aclocal.m4	2009-10-28 14:28:30.000000000 +0100
@@ -396,7 +396,7 @@
 AC_DEFUN([BRLTTY_SUMMARY_END], [dnl
 AC_CONFIG_COMMANDS([brltty-summary],
    [AC_MSG_NOTICE([${brltty_summary_lines}])],
-   [brltty_summary_lines="${brltty_summary_lines}"]
+   [brltty_summary_lines='${brltty_summary_lines}']
 )])
 
 AC_DEFUN([BRLTTY_SUMMARY_ITEM], [dnl
